apps/passport-server/test/database.spec.ts

```diff
@@ -241,4 +249,59 @@ describe("database reads and writes", function () {
     const deletedCommitment = await fetchCommitment(db, email);
     expect(deletedCommitment).to.eq(null);
   });
+
+  step("should be able to interact with the cache", async function () {
+    // insert a bunch of old entries
+    const oldEntries: CacheEntry[] = [];
+    for (let i = 0; i < 20; i++) {
+      oldEntries.push(await setCacheValue(db, "i_" + i, i + ""));
+    }
+    await sqlQuery(
+      db,
+      `
+    update cache
+      set time_created = NOW() - interval '5 days',
+      time_updated = NOW() - interval '5 days'
+    `
+    );
+
+    // old entries inserted, let's insert some 'new' ones
+    await setCacheValue(db, "key", "value");
+    const firstEntry = await getCacheValue(db, "key");
+    expect(firstEntry?.cache_value).to.eq("value");
+    await setCacheValue(db, "key", "value2");
+    const editedFirstEntry = await getCacheValue(db, "key");
+    expect(editedFirstEntry?.cache_value).to.eq("value2");
+    expect(editedFirstEntry?.time_created?.getTime()).to.eq(
+      firstEntry?.time_created?.getTime()
+    );
+
+    await setCacheValue(db, "spongebob", "squarepants");
+    const spongebob = await getCacheValue(db, "key");
+    expect(spongebob?.cache_value).to.eq("value2");
```

Comment:
Was this meant to be `getCacheValue(db, "spongebob")`?

Reply:
yea
![image](https://github.com/proofcarryingdata/zupass/assets/2636237/6b67b5b1-b087-404d-96ac-ec6e48a7e826)